crates/typst/src/layout/grid/mod.rs

```diff
@@ -36,9 +36,18 @@ use crate::visualize::{Paint, Stroke};
 /// There are multiple sizing modes for columns and rows that can be used to
 /// create complex layouts.
 ///
-/// The sizing of the grid is determined by the track sizes specified in the
-/// arguments. Because each of the sizing parameters accepts the same values, we
-/// will explain them just once, here. Each sizing argument accepts an array of
+/// While the grid and table elements work very similarly, they are intended for
+/// different use cases and carry different semantics. The grid is intended for
```

Comment:
```suggestion
/// different use cases and carry different semantics. The grid element is intended for
```
Maybe it's just me, but "the grid" alone feels weird.